mudlet-mapper.xml

```diff
@@ -5662,6 +5665,18 @@ function mmp.addWingsAchaea()
         mmp.tempSpecialExit(51188, getcmd(&quot;duanathar&quot;))
       end
     end
+		if
+    	mmp.settings.duanatharic and
+    	gmcp.Room and
+    	not table.contains(gmcp.Room.Info.details, &quot;indoors&quot;)
+  	then
+	  	--Island wings.
+    	if mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Main&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Arcadia&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Outer&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Meropis&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Island&quot;) or mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;North&quot;) then
+      	if mmp.settings.duanatharic then
```

Comment:
You don't need to re-check for `mmp.settings.duanatharic` here, because it's already covered in the outer `if`.